app/Http/Controllers/TodoController.php

```diff
@@ -9,15 +9,28 @@
 class TodoController extends Controller
 {
 
-    public function index() : JsonResponse
+    public function index(): JsonResponse
     {
         $todos = Todo::all();
         return response()->json($todos);
     }
 
-    public function store(Request $request)
+    public function store(Request $request): JsonResponse
     {
-
+        $validatedData = $request->validate([
+            'title' => 'required|string|max:255',
+            'user_id' => 'required|integer',
+            'priority' => 'string|in:Low,Medium,High',
+            'description' => 'string',
+            'is_checked' => 'boolean',
+            'due_date' => 'date',
+        ]);
```

Comment:
- A better approach is to use Form request validation https://laravel.com/docs/10.x/validation#form-request-validation
- Use an array, not pipes, ex 'user_id' => ['required', 'integer'] is much more readable